src/SFA.DAS.FindEpao/Application/Courses/Services/CourseEpaoIsValidFilterService.cs

```diff
@@ -38,5 +38,23 @@ public bool ValidateEpaoStandardDates(DateTime? dateStandardApprovedOnRegister,
 
             return true;
         }
+
+        public bool ValidateVersionDates(DateTime? effectiveFrom, DateTime? effectiveTo)
+        {
+
+            if ((effectiveFrom.HasValue) && (effectiveTo.HasValue))
```

Comment:
Mileage may vary but this can be shortened 
`public bool ValidateVersionDates(DateTime? effectiveFrom, DateTime? effectiveTo) =>
            ((effectiveFrom.HasValue && effectiveFrom.Value <= DateTime.UtcNow)
                && (!effectiveTo.HasValue || effectiveTo.Value >= DateTime.UtcNow));         `

Granted yours is more readable but i'd think you could combine the two if's into one.

Reply:
agreed - modified